### DISCUSSION TITLE
Recebimento de equity

### DISCUSSION THREAD

**Discussion by brenoarosa** [DyegoCosta/trabalhando-remoto#169] [General]
Pessoal, pergunta bem aberta mas mais pra saber se alguém já teve experiência e queira contar sobre.
Vou começar numa empresa dos EUA daqui a 1 semana. Contrato por CNPJ bem padrão como a maior parte da galera daqui.
A oferta tem um pacote de equity (opção de compra de ações )e ainda tenho algumas dúvidas.
Não sei se pelo contrato ser com o CNPJ o pacote de equity tb fica associado ao CNPJ, precisando depois ser transferido pra pessoa física (e possíveis burocracias que isso envolve).
Também tenho dúvidas sobre as regras de taxações americanas, que cobram o "lucro" na conversão da opção em ação mesmo antes de realizar uma venda valem também pra quem tá no Brasil ou seguimos as regras de opções daqui? Muito do material de posts sobre equity nos EUA toca nessas regras de taxação e chegam a conclusões que podem ser diferentes se nós não precisarmos segui-las.

Tem algo de especial pra ficar atento nesses casos?

Desculpem trazer o tema tão aberto ainda mas preferi já abrir aqui. Pretendo conversar com o contador sobre e tendo mais informações de interesse geral vou compartilhando na issue.

**Comment by fforbeck** [comment]
Opa @brenoarosa 

Tem uma série de implicações se você deixar o pacote de equity associado ao CNPJ. Sugiro entrar em contato com algum advogado especialista para te ajudar a analisar a proposta de equity.  No meu caso eu pensava que ficaria mais barato para trazer a grana depois de uma eventual venda, porém só vale a pena se a empresa for no Lucro Real. O pessoal da https://www.ped.adv.br/en/ me ajudou bastante (procure pela Beatriz). Basicamente o meu o contrato ficou na PJ e o pacote de equity na PF. Você provavelmente precisará preencher um form falando que é estrangeiro e recolhe impostos em outro país. No Brasil ainda é muito nebulosa essa questão da taxação de equity. Tem lugar que fala que você já deve ser taxado assim que recebe as opções, porém há decisões que falam que a taxação deve ocorrer somente após a venda das opções (o que é mais lógico). É bem zuado, por isso sugiro fortemente um auxílio jurídico aqui. É importante pensar que isso tem que estar bem estruturado para que no futuro você não tenha problemas em caso de venda das opções. O pessoal tem escritório nos EUA também, então eles conseguem analizar a melhor solução para você levando em consideração o tipo do pacote equity, se você tem planos de mudar para os EUA, etc.

**Comment by brenoarosa** [comment]
Valeu demais pelo relato!
To esperando chegar o contrato de equity que vem separado da offer mas acho que vou procurar um advogado pra auxiliar mesmo pra ter uma segurança maior. Tem bastante informação sobre equity mas pra americano/residente nos EUA, não consegui achar nada pra quem tá em situação parecida com a nossa.
Obrigado pela dica de escritório, não sei se é ok perguntar mas se puder sabe dizer mais ou menos quanto foi esse serviço de assessoria pro contrato?

**Comment by fforbeck** [comment]
> Valeu demais pelo relato! To esperando chegar o contrato de equity que vem separado da offer mas acho que vou procurar um advogado pra auxiliar mesmo pra ter uma segurança maior. Tem bastante informação sobre equity mas pra americano/residente nos EUA, não consegui achar nada pra quem tá em situação parecida com a nossa. Obrigado pela dica de escritório, não sei se é ok perguntar mas se puder sabe dizer mais ou menos quanto foi esse serviço de assessoria pro contrato?

Lembro que foi algo em torno de 3.5k. Mas depende muito do que você precisa. Você consegue agendar uma call com eles só para conversar e entender melhor como eles trabalham, cobram e como eles podem te ajudar. Acho que vale a pena.

**Comment by brenoarosa** [comment]
show, obrigado demais cara, ajudou bastante!
Vou conferir com alguns conhecidos em situação parecida pra ver se consigo mais indicações de escritórios tb.
